relay/README.md

```diff
@@ -73,106 +52,156 @@ Cast:
 - QmRelay, a node which speaks the circuit relay protocol (go-ipfs or js-ipfs).
 
 Scene 1:
-- QmOne wants to connect to QmTwo,
-  and through peer routing has acquired a set of addresses of QmTwo.
+- QmOne wants to connect to QmTwo, and through peer routing has acquired a set of addresses of QmTwo.
 - QmTwo doesn't support any of the transports used by QmOne.
 - Awkward silence.
 
 Scene 2:
 - All three nodes have learned to speak the `/ipfs/relay/circuit` protocol.
 - QmRelay is configured to allow relaying connections between other nodes.
 - QmOne is configured to use QmRelay for relaying.
-- QmOne automatically added `/p2p-circuit/ipfs/QmTwo` to its set of QmTwo addresses.
+- QmOne automatically added `/p2p-circuit/p2p/QmTwo` to its set of QmTwo addresses.
 - QmOne tries to connect via relaying, because it shares this transport with QmTwo.
 - A lively and prolonged dialogue ensues.
 
-
 ## Addressing
 
-`/p2p-circuit` multiaddrs don't carry any meaning of their own.
-They need to encapsulate a `/p2p` address, or
-be encapsulated in a `/p2p` address, or both.
+`/p2p-circuit` multiaddrs don't carry any meaning of their own. They need to encapsulate a `/p2p` address, or be encapsulated in a `/p2p` address, or both.
```

Comment:
The case that is not allowed is `<relay peer multiaddr>/p2p-circuit/` (destination omitted). What about the following wording?

> A `/p2p-circuit` multiaddress needs to encapsulate a `/p2p` destination address, and optionally be encapsulated by a `/p2p` relay address.